pkg/cloud-controller-manager/loadBalancer.go

```diff
@@ -70,18 +67,8 @@ func (l *LoadBalancerManager) GetLoadBalancer(ctx context.Context, clusterName s
 }
 
 func (l *LoadBalancerManager) GetLoadBalancerName(ctx context.Context, clusterName string, service *v1.Service) string {
```

Comment:
Remove the clusterName param

Reply:
We can not do that because it is a method of cloudpropvider.LoadBalancer interface.